app/controllers/api/v1/interactive_pages_controller.rb

```diff
@@ -59,7 +59,15 @@ def create_page
     authorize! :update, activity
     return error("Can't find activity #{params[:activity_id]}") unless activity
     activity.pages.create()
+    last_idx = activity.pages.length - 1
+    prev_last_idx = activity.pages.length - 2
+    if activity.pages[prev_last_idx].is_completion
+      activity.pages[prev_last_idx].position = last_idx
+      activity.pages[prev_last_idx].move_lower
```

Comment:
As I understand it `move_lower` was deprecated as of Rails 2. I'm not sure it matters, but it may be better to find a different way to do this.

Reply:
I got it from here? https://github.com/brendon/acts_as_list

Reply:
Hm. Maybe it's not the same thing I was looking at then which is [this](https://apidock.com/rails/ActiveRecord/Acts/List/InstanceMethods/move_lower). I see you're discussing acts_as_list with Noah in Slack, so I'll defer to the outcome of that conversation about this.